avm/res/virtual-machine-images/image-template/tests/e2e/max/dependencies.bicep

```diff
@@ -271,11 +271,11 @@ resource assetsStorageAccount_upload 'Microsoft.Resources/deploymentScripts@2023
     scriptContent: loadTextContent('../../../../../../utilities/e2e-template-assets/scripts/Set-StorageContainerContentByEnvVar.ps1')
     environmentVariables: [
       {
-        name: 'script_Install__LinuxPowerShell_sh' // May only be alphanumeric characters & underscores. The upload will replace '_' with '.' and '__' with '-'.
+        name: '__SCRIPT__Install__LinuxPowerShell_sh' // May only be alphanumeric characters & underscores. The upload will replace '_' with '.' and '__' with '-'.
```

Comment:
Is the updated script used by other modules as well (in other words: should this update happen in other module tests as well)?

Reply:
Hey @eriqua, only by this module - and the to-be AIB pattern module.